request.proto

```diff
@@ -165,7 +165,7 @@ message NearestStopPointsRequest {
 
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
-    required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```

Comment:
still really not fan of the name (but the problem is more on params) and it should be `durations` at least, no? `duration_bounds` would be nicer I think.